src/components/questions/AudioRecorder.tsx

```diff
@@ -0,0 +1,199 @@
+import AudioRecorderPlayer, {
+  AVEncoderAudioQualityIOSType,
+  AVEncodingOption,
+  AudioEncoderAndroidType,
+  AudioSet,
+  AudioSourceAndroidType,
+  PlayBackType,
+} from 'react-native-audio-recorder-player';
+import { PermissionsAndroid, Platform, StyleSheet, TouchableOpacity, View } from 'react-native';
+import React, { Component } from 'react';
+
+import Icon from 'react-native-vector-icons/FontAwesome';
+import COLORS from '../../assets/colors';
+
+interface MyProps {
+  isPlaying: boolean;
+  isRecording: boolean;
+  recordUri: string;
+  isVisible: boolean;
+}
+class AudioRecorder extends Component<
+  { onVoiceSelected: (recordUri: string) => void } | {},
+  MyProps
+> {
+  private audioRecorderPlayer: AudioRecorderPlayer;
+
+  constructor(props: MyProps) {
+    super(props);
+    this.state = {
+      isRecording: false,
+      isPlaying: false,
+      recordUri: '',
+      isVisible: false,
```

Comment:
Wat is hier het idee van?

Reply:
isVisible had ik voor de visibility van sommige icons, ik kan het wel renamen naar isDisabled als je dat fijner vind?